packages/frontend/src/components/UserSettings/UserAttributesPanel/index.tsx

```diff
@@ -0,0 +1,243 @@
+import { subject } from '@casl/ability';
+import { OrgAttribute } from '@lightdash/common';
+import {
+    Button,
+    Flex,
+    Group,
+    Modal,
+    Stack,
+    Table,
+    Text,
+    Title,
+    Tooltip,
+} from '@mantine/core';
+import {
+    IconAlertCircle,
+    IconCircleX,
+    IconEdit,
+    IconInfoCircle,
+} from '@tabler/icons-react';
+import { FC, useState } from 'react';
+import { useOrganization } from '../../../hooks/organization/useOrganization';
+import { useTableStyles } from '../../../hooks/styles/useTableStyles';
+import {
+    useUserAttributes,
+    useUserAttributesDeleteMutation,
+} from '../../../hooks/useUserAttributes';
+import { useApp } from '../../../providers/AppProvider';
+import LoadingState from '../../common/LoadingState';
+import MantineIcon from '../../common/MantineIcon';
+import { SettingsCard } from '../../common/Settings/SettingsCard';
+import ForbiddenPanel from '../../ForbiddenPanel';
+import UserAttributeModal from './UserAttributeModal';
+
+const UserListItem: FC<{
+    orgUserAttribute: OrgAttribute;
+    allUserAttributes: OrgAttribute[];
+}> = ({ orgUserAttribute, allUserAttributes }) => {
+    const [isDeleteDialogOpen, setIsDeleteDialogOpen] = useState(false);
+    const [isEditDialogOpen, setIsEditDialogOpen] = useState(false);
```

Comment:
there's this hook that can be handful here https://mantine.dev/hooks/use-disclosure/

Reply:
very cool